Update content for introduction.md (#295)
akeneo/pim-api-docs: content/guides/dam-connection/introduction.md

![Macro overview of the DAM connection](../../img/guides/dam-connection-macro.svg)

In this guide, we consider the DAM as the master of the product assets. It means that assets are sent from the DAM to the PIM, and not the other way around.  
First, photographers or marketers upload their new product assets into the DAM. Then, thanks to the connector we're going to build in this guide, these assets are sent via the API to the Asset Manager of our PIM.  

> Once assets are inside the Asset Manager, there are two ways to link them to the right product:

Reviewer: ```suggestion
Once the assets are in the Asset Manager, there are two ways to link them to the right product:
```

Now, let's take a closer look. 

> You already have a DAM installed and a lot of your assets are already stored in it. Some of these assets are product assets: product user guides, product images, product video shootings,...  

Reviewer: ```suggestion
You already have a DAM installed and a lot of your assets are already stored in it. Some of those assets are product assets: product user guides, product images, product video shootings,...  
```

> You also have at your disposal an Akeneo PIM. And your goal is quite simple. You want the product assets located in your DAM to be sent to your PIM, so that you can have all the visuals, documentation and product shootings of your product along with your product information in one single product sheet.

Reviewer: ```suggestion
You are also using an Akeneo PIM. And your goal is quite simple. You want the product assets located in your DAM to be sent to your PIM, so that you can have all the visuals, documentation and product shootings of your product along with your product information in one single product sheet.
```

> 1. First, you're going to need to understand how the Asset Manager works. Carefully read about all its concepts, right [here](/documentation/asset-manager.html), and you'll become an Asset Manager master! On a more serious note, these concepts are key to create a great connector that will perfectly fit with our PIM. 😉

Reviewer: ```suggestion
1. First, you're going to need to understand how the Asset Manager works. Carefully read about all its concepts, right [here](/documentation/asset-manager.html), and you'll become an Asset Manager master! On a more serious note, those concepts are key to creating a great connector that will perfectly fit with our PIM. 😉
```

2. Then, we strongly encourage you to read the [glossary](glossary.html). It will help you grasp the basic notions we are going to use throughout this guide. It's worth the read - you'll see it's pretty short.

> 3. Before beginning to write any code, you'll need to study carefully your DAM and its asset modelisation. It's a crucial step, don't miss it. You'll find more about how to successfully initialize your Asset Manager families that will match the assets you're going to receive from the DAM, in the [Structure your DAM and PIM](pre-requisites.html) section.

Reviewer: ```suggestion
3. Before beginning to write any code, you'll need to carefully study your DAM and its asset modelization. It's a crucial step, don't miss it. You'll find more about how to successfully initialize your Asset Manager families that will match the assets you're going to receive from the DAM, in the [Structure your DAM and PIM](pre-requisites.html) section.
```

> 4. For this step, you are going to choose your connector architecture and technical stack. Check out this [section](technical-stack.html) to learn more. You'll see it also provides you with pro tips regarding the scalability or the error management and so on...

Reviewer: ```suggestion
4. For this step, you are going to choose your connector's architecture and technical stack. Check out this [section](technical-stack.html) to find out more. You'll see it also provides pro tips regarding scalability, error management and so on...
```

> 5. Now, hands on your keyboard, it's the part where you start coding. Build the DAM-PIM connector by following the steps described in the [Synchronize assets](synchronize-assets.html) section.

Reviewer: ```suggestion
5. Now, hands on your keyboard, this is the part where you start coding. Build the DAM-PIM connector by following the steps described in the [Synchronize assets](synchronize-assets.html) section.
```

You'll find it [here](https://github.com/akeneo/dam-connector). Enjoy!

::: warning

> This connector skeleton is provided as is. It is only supported in **best effort** mode, either you are an Enterprise or Community user.

Reviewer: ```suggestion
This connector skeleton is provided as-is. It is only supported in **best effort** mode, whether you are an Enterprise or a Community user.
```

:::

::: warning
This skeleton is **far from being production-ready**. We don't recommend you to use it as is. Its sole purpose is for training and learning.
:::

If you ever find a bug, do not hesitate to directly open an issue on the [Github](https://github.com/akeneo/dam-connector/issues) repository.

> Don't forget that we also love it when you contribute and help us make better tools. So do not hesitate! Each feedback/pull request is very welcome!

Reviewer: ```suggestion
Don't forget that we also love it when you contribute and help us make better tools. So do not hesitate! Every feedback/pull request is very welcome!
```
